Package.swift

```diff
@@ -4,20 +4,20 @@
 import PackageDescription
 
 let package = Package(
-    name: "tkey_pkg",
+    name: "tkey-mpc-swift",
     platforms: [
-        .iOS(SupportedPlatform.IOSVersion.v14),
+        .iOS(.v13), .macOS(.v10_15)
     ],
     products: [
         // Products define the executables and libraries a package produces, and make them visible to other packages.
         .library(
-            name: "ThresholdKey",
-            targets: ["tkey-pkg"]),
+            name: "tkey-mpc-swift",
+            targets: ["tkey-mpc-swift"]),
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
         // .package(url: /* package url */, from: "1.0.0"),
-        .package(name: "TorusUtils", url: "https://github.com/torusresearch/torus-utils-swift" , from: "6.1.0"),
+        .package(name: "TorusUtils", url: "https://github.com/torusresearch/torus-utils-swift" , .branch("extension")),
```

Comment:
This needs to be updated to a version when https://github.com/torusresearch/torus-utils-swift/pull/79 merges